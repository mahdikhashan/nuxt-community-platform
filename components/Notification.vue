<template>
  <div
    class="
      flex
      bg-gray-200
      items-center
      justify-between
      p-4
      rounded-md
      border-2
    "
  >
    <div class=""><slot></slot></div>
    <a class="">
      <img
        src="data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAABmJLR0QA/wD/AP+gvaeTAAAAnElEQVRIie2VSw6DMAwFB+7QCC5Y7guoSPQw6caRgFapP7DjbcmbIUg2cMeRJ5AcvSTdagYgA5NRkoBZukPt4AMY5eAC9Er4tOl0/woWiRlukbjhGkkYXpOcBv8lWQV6GnwreQk4A28tvA1Im0B3l+MnKjfRzokaXoCeYVTDNc/C8LDEUjRLPG9l6pR1PWNf12W6q+saLv7h3PnKB/1VUtVNViCxAAAAAElFTkSuQmCC"
      />
    </a>
  </div>
</template>

<script>
export default {
  name: "Notification",
};
</script>
