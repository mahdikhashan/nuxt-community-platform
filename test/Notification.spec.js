import { mount } from '@vue/test-utils'
import Notification from '@/components/Notification.vue'

describe('Notification', () => {
  test('is a Vue instance', () => {
    const wrapper = mount(Notification)
    expect(wrapper.vm).toBeTruthy()
    expect(wrapper.html()).toMatchSnapshot()
  })
})
