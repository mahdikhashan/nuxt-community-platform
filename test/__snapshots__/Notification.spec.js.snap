// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Notification is a Vue instance 1`] = `
"<div class=\\"
    flex
    bg-gray-100
    items-center
    justify-between
    p-4
    rounded-md
    border-[1px]
  \\">
  <div></div> <button class=\\"w-4\\"><img src=\\"data:image/png;base64,iVBORw0KGgoAAAANSUhEUgAAABgAAAAYCAYAAADgdz34AAAABmJLR0QA/wD/AP+gvaeTAAAAnElEQVRIie2VSw6DMAwFB+7QCC5Y7guoSPQw6caRgFapP7DjbcmbIUg2cMeRJ5AcvSTdagYgA5NRkoBZukPt4AMY5eAC9Er4tOl0/woWiRlukbjhGkkYXpOcBv8lWQV6GnwreQk4A28tvA1Im0B3l+MnKjfRzokaXoCeYVTDNc/C8LDEUjRLPG9l6pR1PWNf12W6q+saLv7h3PnKB/1VUtVNViCxAAAAAElFTkSuQmCC\\"></button>
</div>"
`;
